include/psen_scan_v2/ros_scanner_node.h

```diff
@@ -34,7 +34,7 @@ namespace psen_scan_v2
 using namespace std::chrono_literals;
 
 /**
- * @brief ROS Node for fetching and publishing laserscan data from the scanner.
+ * @brief ROS Node that continuesly publishes scan data of a single PSENscan laser scanner.
```

Comment:
```suggestion
 * @brief ROS Node that continuously publishes scan data of a single PSENscan laser scanner.
```

By the way (not a documentation but design question): What does the `T` suffix in `RosScannerNodeT` mean?

Reply:
Without being present at the decision I expect the T stands for Template.
```
typedef ROSScannerNodeT<> ROSScannerNode;

template <typename S>
ROSScannerNodeT<S>
```

I think the template method was mainly used to inject the scanner mock for the integration tests.